src/Model/EditorMetricsReducer.re

```diff
@@ -7,16 +7,14 @@ open Feature_Editor;
 
 let reduce = (v: EditorMetrics.t, action) => {
   switch (action) {
-  | EditorGroupSetSize(_, {pixelWidth, pixelHeight}) => {
-      ...v,
-      pixelWidth,
-      pixelHeight,
-    }
-  | EditorFont(Service_Font.FontLoaded({measuredHeight, measuredWidth, _})) => {
-      ...v,
-      lineHeight: measuredHeight,
-      characterWidth: measuredWidth,
-    }
+  | EditorGroupSetSize(_, {pixelWidth, pixelHeight}) =>
+    EditorMetrics.{pixelWidth, pixelHeight}
```

Comment:
Next step would be to push `pixelWidth` and `pixelHeight` into the editor, so we can remove `EditorMetrics.t` completely.